frontend/src/component/project/Project/EditProject/EditProject.tsx

```diff
@@ -14,28 +14,40 @@ import { useContext } from 'react';
 import AccessContext from 'contexts/AccessContext';
 import { Alert } from '@mui/material';
 import { GO_BACK } from 'constants/navigate';
+import { usePlausibleTracker } from '../../../../hooks/usePlausibleTracker';
 
 const EditProject = () => {
     const { uiConfig } = useUiConfig();
+    const { projectScopedStickiness } = uiConfig.flags;
     const { setToastData, setToastApiError } = useToast();
     const { hasAccess } = useContext(AccessContext);
     const id = useRequiredPathParam('projectId');
     const { project } = useProject(id);
     const navigate = useNavigate();
+    const { trackEvent } = usePlausibleTracker();
+
+    const storedStickiness = localStorage.getItem(`defaultStickiness.${id}`);
 
     const {
         projectId,
         projectName,
         projectDesc,
+        projectStickiness,
         setProjectId,
         setProjectName,
         setProjectDesc,
+        setProjectStickiness,
         getProjectPayload,
         clearErrors,
         validateProjectId,
         validateName,
         errors,
-    } = useProjectForm(id, project.name, project.description);
+    } = useProjectForm(
+        id,
+        project.name,
+        project.description,
+        storedStickiness != null ? storedStickiness : undefined
```

Comment:
wouldn't storedStickiness on its own work without the ternary?

Reply:
storedStickiness can be null from localStorage - ts doesn't like that here